app/http.php

```diff
@@ -1015,6 +1016,24 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
                 ];
             };
 
+            $pingStart = \microtime(true);
+            while (true) {
```

Comment:
How is this different than what's happening here?? Why do we need to two very similar blocks? can we be more strategic/minimalistic? 

https://github.com/open-runtimes/executor/blob/6b41784bbdfb9e3cfaeb53532f693740a91bcebd/app/http.php#L828-L839

Reply:
The new check actually tests E2E communication. Status is okay once Docker container starts. But server is only available to receive requests on port some time later. Depending on user's code, this may be long time.

Reply:
We have multiple check and as far as I can tell, we need them all. It will be simplified when we split endpoint's responsibility and do multiple requests from Client (Appwrite). Until then, this looks fine to me. I cant find any easy way to simplify.